docs/src/rsh/appinit/index.md

```diff
@@ -114,6 +121,9 @@ Each function must occur exactly once in the entire program.
 
 ### {#ref-programs-appinit-view} View Definition
 
+Views generate a funtion that anyone can call to learn the state without being tied to the implementation.
+On Ethereum and Conflux, `view` can call contract-to-contract and API functions that are guaranteed to be read-only.
```

Comment:
I don't know what you're getting at here.

Reply:
I wrote that based on information I found in our documentation and online, but I tried to search for it on our documentation, and I am not finding it. If it doesn't make sense to you, I think that is a good clue that I missed the mark and we should not include it. I will remove it.

Reply:
It is true that they are part of the on-chain API and that they are read-only, but the phrasing of this is confusing to me.